### ISSUE TITLE
Build failure 0278b57

### ISSUE THREAD

**Issue by ms6676749** [CDrummond/cantata#813]
System: Archlinux
CMake options:
    -DCMAKE_INSTALL_PREFIX=/usr \
    -DCMAKE_BUILD_TYPE=Debug \
    -DENABLE_KDE=OFF \
    -DENABLE_QT5=ON \
    -DENABLE_UDISKS2=ON \
    -DENABLE_LIBVLC=OFF

Cantata failed to compile when updating from bf353b1 to 0278b57.  It looks like an ffmpeg/replaygain issue, as ffmpeg on my system just updated to 1.3.0 prior to the start of the compile.

```
==> Starting build()...
-- The C compiler identification is GNU 5.3.0
-- The CXX compiler identification is GNU 5.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Taglib found: -L/usr/lib -ltag
-- Performing Test TAGLIB_CAN_SAVE_ID3VER
-- Performing Test TAGLIB_CAN_SAVE_ID3VER - Success
-- Found Cdparanoia: /usr/lib64/libcdda_paranoia.so;/usr/lib64/libcdda_interface.so;-lm  
-- Performing Test CDPARANOIA_HAS_CACHEMODEL_SIZE
-- Performing Test CDPARANOIA_HAS_CACHEMODEL_SIZE - Success
-- Found CDDB: /usr/include  
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29") 
-- Checking for one of the modules 'libmusicbrainz5cc'
-- Checking for one of the modules 'libmusicbrainz5'
-- Found MUSICBRAINZ5: /usr/include/neon;/usr/include/libxml2  
-- Performing Test TAGLIB_ASF_FOUND
-- Performing Test TAGLIB_ASF_FOUND - Success
-- Performing Test TAGLIB_MP4_FOUND
-- Performing Test TAGLIB_MP4_FOUND - Success
-- Taglib-Extras found: -L/usr/lib -ltag-extras
-- Found FFMPEG: /usr/lib64/libavcodec.so  
-- Found MPG123: /usr/lib64/libmpg123.so  
-- Checking for module 'libmtp'
--   Found libmtp, version 1.1.11
-- Found MTP: /usr/lib64/libmtp.so
-- Found ZLIB: /usr/lib64/libz.so (found version "1.2.8") 
-- Enabling exceptions
-- Using supplied libebur128
-- Found SPEEXDSP: /usr/lib64/libspeexdsp.so  
-- Performing Test HAS_SSE3_EXTENSIONS
-- Performing Test HAS_SSE3_EXTENSIONS - Success
-- Found UDev: /usr/include  
-- Building Solid UDev backend.
-- Building Solid HAL backend.
-- Building Solid UDisks2 backend.
-- Building Qt5 version

-----------------------------------------------------------------------------
-- The following external packages were located on your system.
-- This installation will have the extra features provided by these packages.
-----------------------------------------------------------------------------
   * TagLib - Tag editor, file organiser, etc.
   * libcdda_paranoia - CD ripping.
   * libcddb - CD info retrieval via CDDB.
   * libmusicbrainz5 - CD info retrieval via MusicBrainz.
   * TagLib-Extras - Tag editor, file organiser, etc.
   * libavcodec/libavutil/libavformat - ReplayGain calculation.
   * libmpg123 - ReplayGain calculation.
   * libmtp - MTP Device Support.
   * Qt5DBus - DBus support.
   * Qt5Multimedia - MPD HTTP stream playback.
   * SpeexDSP - Used to speedup/improve ReplayGain calculation.
   * UDev - UDev support for Solid
   * media-player-info - Enables identification and querying of portable media players

-----------------------------------------------------------------------------
-- Congratulations! All external packages have been found.
-----------------------------------------------------------------------------

-- Configuring done
-- Generating done
-- Build files have been written to: /home/user/Projects/cantata-git/src/build
Scanning dependencies of target ebur128
[  0%] Generating moc_qtiocompressor.cpp
[  0%] Generating moc_thread.cpp
Scanning dependencies of target translations
[  0%] Generating moc_fancytabwidget.cpp
[  1%] Generating moc_taghelper.cpp
[  1%] Generating ui_shortcutssettingswidget.h
[  1%] Generating backends/udisks2/dbus/moc_manager.cpp
[  1%] Building C object 3rdparty/ebur128/CMakeFiles/ebur128.dir/ebur128.c.o
[  1%] Generating moc_messagewidget.cpp
Removed plural forms as the target language has less forms.
If this sounds wrong, possibly the target language is not set or recognized.
Scanning dependencies of target support-core
Scanning dependencies of target cantata-tags
Scanning dependencies of target qtiocompressor
Updating '/home/user/Projects/cantata-git/src/build/po/cantata_ru.qm'...
    Generated 1118 translation(s) (1118 finished and 0 unfinished)
    Ignored 117 untranslated source text(s)
[  1%] Building CXX object support/CMakeFiles/support-core.dir/moc_thread.cpp.o
[  1%] Building CXX object 3rdparty/qtiocompressor/CMakeFiles/qtiocompressor.dir/qtiocompressor.cpp.o
[  1%] Generating moc_inputdialog.cpp
[  1%] Generating moc_pagewidget.cpp
[  1%] Building CXX object tags/CMakeFiles/cantata-tags.dir/main.cpp.o
[  1%] Generating moc_block.cpp
Updating '/home/user/Projects/cantata-git/src/build/po/cantata_es.qm'...
    Generated 968 translation(s) (968 finished and 0 unfinished)
    Ignored 267 untranslated source text(s)
[  2%] Building CXX object 3rdparty/qtiocompressor/CMakeFiles/qtiocompressor.dir/moc_qtiocompressor.cpp.o
[  2%] Generating moc_deviceinterface.cpp
[  2%] Generating moc_action.cpp
Updating '/home/user/Projects/cantata-git/src/build/po/cantata_zh_CN.qm'...
    Generated 467 translation(s) (467 finished and 0 unfinished)
    Ignored 768 untranslated source text(s)
[  2%] Generating moc_devicemanager_p.cpp
Updating '/home/user/Projects/cantata-git/src/build/po/cantata_cs.qm'...
    Generated 1235 translation(s) (1235 finished and 0 unfinished)
[  2%] Generating moc_actioncollection.cpp
[  2%] Generating moc_devicenotifier.cpp
Updating '/home/user/Projects/cantata-git/src/build/po/cantata_de.qm'...
    Generated 768 translation(s) (768 finished and 0 unfinished)
    Ignored 467 untranslated source text(s)
[  3%] Generating moc_configdialog.cpp
[  3%] Generating moc_device_p.cpp
Updating '/home/user/Projects/cantata-git/src/build/po/cantata_en_GB.qm'...
    Generated 21 translation(s) (21 finished and 0 unfinished)
    Ignored 1214 untranslated source text(s)
[  3%] Generating moc_genericinterface.cpp
[  3%] Generating moc_combobox.cpp
Updating '/home/user/Projects/cantata-git/src/build/po/cantata_pl.qm'...
    Generated 1235 translation(s) (1235 finished and 0 unfinished)
[  3%] Generating moc_portablemediaplayer.cpp
Removed plural forms as the target language has less forms.
If this sounds wrong, possibly the target language is not set or recognized.
Updating '/home/user/Projects/cantata-git/src/build/po/cantata_hu.qm'...
    Generated 1118 translation(s) (1118 finished and 0 unfinished)
    Ignored 117 untranslated source text(s)
[  3%] Generating moc_shortcuthandler.cpp
[  3%] Linking C static library libebur128.a
[  3%] Generating moc_storageaccess.cpp
[  3%] Generating moc_windowmanager.cpp
Updating '/home/user/Projects/cantata-git/src/build/po/cantata_fr.qm'...
    Generated 1027 translation(s) (1027 finished and 0 unfinished)
    Ignored 208 untranslated source text(s)
[  3%] Generating moc_storagedrive.cpp
[  3%] Built target ebur128
[  3%] Building CXX object support/CMakeFiles/support-core.dir/utils.cpp.o
Removed plural forms as the target language has less forms.
If this sounds wrong, possibly the target language is not set or recognized.
[  3%] Building CXX object support/CMakeFiles/support-core.dir/thread.cpp.o
[  4%] Generating moc_pathrequester.cpp
[  4%] Generating moc_opticaldrive.cpp
[  4%] Building CXX object tags/CMakeFiles/cantata-tags.dir/taghelper.cpp.o
Updating '/home/user/Projects/cantata-git/src/build/po/cantata_ko.qm'...
    Generated 1235 translation(s) (1235 finished and 0 unfinished)
[  4%] Built target translations
[  4%] Generating __/support/moc_thread.cpp
[  4%] Generating moc_storagevolume.cpp
[  5%] Generating moc_replaygain.cpp
[  5%] Generating moc_kmessagewidget.cpp
[  5%] Generating moc_opticaldisc.cpp
[  5%] Generating moc_trackscanner.cpp
[  5%] Generating moc_urllabel.cpp
[  5%] Generating ifaces/moc_device.cpp
[  5%] Generating moc_jobcontroller.cpp
[  5%] Generating ifaces/moc_devicemanager.cpp
[  5%] Generating moc_dialog.cpp
Scanning dependencies of target cantata-replaygain
[  5%] Generating backends/shared/moc_rootdevice.cpp
[  5%] Linking CXX static library libqtiocompressor.a
[  5%] Building CXX object replaygain/CMakeFiles/cantata-replaygain.dir/ffmpeginput.cpp.o
[  6%] Generating moc_shortcutsmodel.cpp
[  6%] Building CXX object tags/CMakeFiles/cantata-tags.dir/tags.cpp.o
[  6%] Built target qtiocompressor
[  6%] Building CXX object tags/CMakeFiles/cantata-tags.dir/filetyperesolver.cpp.o
/home/user/Projects/cantata-git/src/cantata-git/replaygain/ffmpeginput.cpp:24:36: fatal error: libavutil/audioconvert.h: No such file or directory
compilation terminated.
replaygain/CMakeFiles/cantata-replaygain.dir/build.make:78: recipe for target 'replaygain/CMakeFiles/cantata-replaygain.dir/ffmpeginput.cpp.o' failed
make[2]: *** [replaygain/CMakeFiles/cantata-replaygain.dir/ffmpeginput.cpp.o] Error 1
CMakeFiles/Makefile2:195: recipe for target 'replaygain/CMakeFiles/cantata-replaygain.dir/all' failed
make[1]: *** [replaygain/CMakeFiles/cantata-replaygain.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[  6%] Generating backends/udev/moc_udevblock.cpp
[  6%] Generating backends/udev/moc_udevdevice.cpp
[  6%] Generating moc_shortcutssettingswidget.cpp
[  6%] Generating moc_keysequencewidget.cpp
[  7%] Generating backends/udev/moc_udevdeviceinterface.cpp
[  7%] Generating backends/udev/moc_udevgenericinterface.cpp
[  7%] Generating moc_acceleratormanager_private.cpp
[  7%] Generating moc_spinner.cpp
[  7%] Generating moc_lineedit.cpp
[  7%] Generating backends/udev/moc_udevmanager.cpp
[  7%] Generating backends/udev/moc_udevportablemediaplayer.cpp
[  7%] Generating backends/shared/moc_udevqt.cpp
[  7%] Building CXX object tags/CMakeFiles/cantata-tags.dir/__/mpd-interface/song.cpp.o
Scanning dependencies of target support
[  7%] Generating backends/hal/moc_halblock.cpp
[  7%] Generating backends/hal/moc_halcdrom.cpp
[  7%] Generating backends/hal/moc_haldevice.cpp
[  7%] Building CXX object support/CMakeFiles/support.dir/moc_fancytabwidget.cpp.o
[  8%] Generating backends/hal/moc_haldeviceinterface.cpp
[  8%] Generating backends/hal/moc_halgenericinterface.cpp
[  8%] Generating backends/hal/moc_halmanager.cpp
[  8%] Building CXX object tags/CMakeFiles/cantata-tags.dir/moc_taghelper.cpp.o
[  8%] Generating backends/hal/moc_halopticaldisc.cpp
[  8%] Generating backends/hal/moc_halstorageaccess.cpp
[  8%] Generating backends/hal/moc_halportablemediaplayer.cpp
[  8%] Generating backends/hal/moc_halstorage.cpp
[  8%] Generating backends/hal/moc_halvolume.cpp
[  9%] Generating backends/udisks2/moc_udisksblock.cpp
[ 10%] Building CXX object support/CMakeFiles/support.dir/moc_messagewidget.cpp.o
[ 10%] Generating backends/udisks2/moc_udisksdevicebackend.cpp
[ 10%] Generating backends/udisks2/moc_udisksdevice.cpp
[ 10%] Generating backends/udisks2/moc_udisksdeviceinterface.cpp
[ 10%] Generating backends/udisks2/moc_udisksgenericinterface.cpp
[ 10%] Generating backends/udisks2/moc_udisksmanager.cpp
[ 10%] Generating backends/udisks2/moc_udisksopticaldisc.cpp
[ 10%] Generating backends/udisks2/moc_udisksopticaldrive.cpp
[ 11%] Generating backends/udisks2/moc_udisksstorageaccess.cpp
[ 11%] Generating backends/udisks2/moc_udisksstoragedrive.cpp
[ 11%] Generating backends/udisks2/moc_udisksstoragevolume.cpp
[ 11%] Building CXX object support/CMakeFiles/support.dir/moc_inputdialog.cpp.o
[ 11%] Building CXX object support/CMakeFiles/support.dir/moc_pagewidget.cpp.o
[ 11%] Building CXX object support/CMakeFiles/support.dir/moc_action.cpp.o
Scanning dependencies of target solidlite
[ 11%] Building CXX object support/CMakeFiles/support.dir/moc_actioncollection.cpp.o
[ 12%] Linking CXX static library libsupport-core.a
[ 12%] Built target support-core
[ 12%] Building CXX object support/CMakeFiles/support.dir/moc_configdialog.cpp.o
[ 12%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/solidnamespace.cpp.o
[ 12%] Building CXX object support/CMakeFiles/support.dir/moc_combobox.cpp.o
[ 12%] Building CXX object support/CMakeFiles/support.dir/moc_shortcuthandler.cpp.o
[ 12%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/managerbase.cpp.o
[ 13%] Building CXX object support/CMakeFiles/support.dir/moc_windowmanager.cpp.o
[ 13%] Building CXX object support/CMakeFiles/support.dir/moc_pathrequester.cpp.o
[ 13%] Building CXX object support/CMakeFiles/support.dir/moc_kmessagewidget.cpp.o
[ 13%] Building CXX object support/CMakeFiles/support.dir/moc_urllabel.cpp.o
[ 13%] Building CXX object support/CMakeFiles/support.dir/moc_dialog.cpp.o
[ 13%] Building CXX object support/CMakeFiles/support.dir/moc_shortcutsmodel.cpp.o
[ 13%] Building CXX object support/CMakeFiles/support.dir/moc_shortcutssettingswidget.cpp.o
[ 13%] Building CXX object support/CMakeFiles/support.dir/moc_keysequencewidget.cpp.o
[ 14%] Building CXX object support/CMakeFiles/support.dir/moc_spinner.cpp.o
[ 14%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/device.cpp.o
[ 14%] Building CXX object support/CMakeFiles/support.dir/moc_acceleratormanager_private.cpp.o
[ 14%] Building CXX object support/CMakeFiles/support.dir/moc_lineedit.cpp.o
[ 14%] Building CXX object support/CMakeFiles/support.dir/icon.cpp.o
[ 14%] Building CXX object support/CMakeFiles/support.dir/fancytabwidget.cpp.o
[ 14%] Building CXX object support/CMakeFiles/support.dir/messagewidget.cpp.o
[ 14%] Building CXX object support/CMakeFiles/support.dir/buddylabel.cpp.o
[ 15%] Linking CXX executable cantata-tags
[ 15%] Building CXX object support/CMakeFiles/support.dir/action.cpp.o
[ 15%] Built target cantata-tags
[ 15%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/devicemanager.cpp.o
[ 16%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/deviceinterface.cpp.o
[ 17%] Building CXX object support/CMakeFiles/support.dir/actioncollection.cpp.o
[ 17%] Building CXX object support/CMakeFiles/support.dir/lineedit.cpp.o
[ 17%] Building CXX object support/CMakeFiles/support.dir/configuration.cpp.o
[ 17%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/genericinterface.cpp.o
[ 17%] Building CXX object support/CMakeFiles/support.dir/gtkstyle.cpp.o
[ 17%] Building CXX object support/CMakeFiles/support.dir/spinner.cpp.o
[ 17%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/block.cpp.o
[ 17%] Building CXX object support/CMakeFiles/support.dir/messagebox.cpp.o
[ 17%] Building CXX object support/CMakeFiles/support.dir/inputdialog.cpp.o
[ 17%] Building CXX object support/CMakeFiles/support.dir/thread.cpp.o
[ 17%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/storagedrive.cpp.o
[ 17%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/opticaldrive.cpp.o
[ 18%] Building CXX object support/CMakeFiles/support.dir/squeezedtextlabel.cpp.o
[ 18%] Building CXX object support/CMakeFiles/support.dir/proxystyle.cpp.o
[ 18%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/storagevolume.cpp.o
[ 18%] Building CXX object support/CMakeFiles/support.dir/touchproxystyle.cpp.o
[ 18%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/opticaldisc.cpp.o
[ 18%] Building CXX object support/CMakeFiles/support.dir/pagewidget.cpp.o
[ 18%] Building CXX object support/CMakeFiles/support.dir/combobox.cpp.o
[ 18%] Building CXX object support/CMakeFiles/support.dir/configdialog.cpp.o
[ 18%] Building CXX object support/CMakeFiles/support.dir/monoicon.cpp.o
[ 18%] Building CXX object support/CMakeFiles/support.dir/gtkproxystyle.cpp.o
[ 18%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/storageaccess.cpp.o
[ 19%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/portablemediaplayer.cpp.o
[ 20%] Building CXX object support/CMakeFiles/support.dir/shortcuthandler.cpp.o
[ 20%] Building CXX object support/CMakeFiles/support.dir/pathrequester.cpp.o
[ 20%] Building CXX object support/CMakeFiles/support.dir/windowmanager.cpp.o
[ 20%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/predicate.cpp.o
[ 20%] Building CXX object support/CMakeFiles/support.dir/kmessagewidget.cpp.o
[ 20%] Building CXX object support/CMakeFiles/support.dir/dialog.cpp.o
[ 20%] Building CXX object support/CMakeFiles/support.dir/shortcutsmodel.cpp.o
[ 20%] Building CXX object support/CMakeFiles/support.dir/shortcutssettingswidget.cpp.o
[ 20%] Building CXX object support/CMakeFiles/support.dir/keysequencewidget.cpp.o
[ 21%] Building CXX object support/CMakeFiles/support.dir/acceleratormanager.cpp.o
[ 21%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/predicateparse.cpp.o
[ 21%] Building CXX object support/CMakeFiles/support.dir/urllabel.cpp.o
[ 21%] Building C object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/predicate_lexer.c.o
[ 21%] Building C object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/predicate_parser.c.o
[ 21%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/xdgbasedirs.cpp.o
[ 21%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/ifaces/block.cpp.o
[ 21%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/ifaces/opticaldrive.cpp.o
[ 22%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/ifaces/device.cpp.o
[ 22%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/ifaces/deviceinterface.cpp.o
[ 22%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/ifaces/devicemanager.cpp.o
[ 22%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/ifaces/genericinterface.cpp.o
[ 22%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/ifaces/opticaldisc.cpp.o
[ 22%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/ifaces/portablemediaplayer.cpp.o
[ 22%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/ifaces/storagedrive.cpp.o
[ 23%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/ifaces/storagevolume.cpp.o
[ 23%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/ifaces/storageaccess.cpp.o
[ 23%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/shared/rootdevice.cpp.o
[ 23%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udev/udevdevice.cpp.o
[ 23%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udev/udevmanager.cpp.o
[ 23%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udev/udevdeviceinterface.cpp.o
[ 23%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udev/udevgenericinterface.cpp.o
[ 23%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udev/udevportablemediaplayer.cpp.o
[ 24%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udev/udevblock.cpp.o
[ 24%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/shared/udevqtclient.cpp.o
[ 24%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/shared/udevqtdevice.cpp.o
[ 24%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/halblock.cpp.o
[ 24%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/halcdrom.cpp.o
[ 24%] Linking CXX static library libsupport.a
[ 24%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/haldeviceinterface.cpp.o
[ 24%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/halfstabhandling.cpp.o
[ 24%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/halgenericinterface.cpp.o
[ 25%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/haldevice.cpp.o
[ 25%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/halmanager.cpp.o
[ 25%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/halopticaldisc.cpp.o
[ 25%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/halportablemediaplayer.cpp.o
[ 25%] Built target support
[ 25%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/halstorageaccess.cpp.o
[ 25%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/halstorage.cpp.o
[ 25%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/halvolume.cpp.o
[ 25%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/udisksmanager.cpp.o
[ 26%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/udisksdevice.cpp.o
[ 26%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/udisksdevicebackend.cpp.o
[ 26%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/udisksblock.cpp.o
[ 26%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/udisksstoragevolume.cpp.o
[ 26%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/udisksdeviceinterface.cpp.o
[ 26%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/udisksopticaldisc.cpp.o
[ 26%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/udisksopticaldrive.cpp.o
[ 26%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/udisksstoragedrive.cpp.o
[ 27%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/udisksstorageaccess.cpp.o
[ 27%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/udisksgenericinterface.cpp.o
[ 27%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/dbus/manager.cpp.o
[ 27%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/moc_block.cpp.o
[ 27%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/moc_deviceinterface.cpp.o
[ 27%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/moc_devicemanager_p.cpp.o
[ 27%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/moc_devicenotifier.cpp.o
[ 27%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/moc_device_p.cpp.o
[ 28%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/moc_genericinterface.cpp.o
[ 28%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/moc_portablemediaplayer.cpp.o
[ 28%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/moc_storageaccess.cpp.o
[ 28%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/moc_storagedrive.cpp.o
[ 28%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/moc_opticaldrive.cpp.o
[ 28%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/moc_storagevolume.cpp.o
[ 28%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/moc_opticaldisc.cpp.o
[ 28%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/ifaces/moc_device.cpp.o
[ 29%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/ifaces/moc_devicemanager.cpp.o
[ 29%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/shared/moc_rootdevice.cpp.o
[ 29%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udev/moc_udevblock.cpp.o
[ 29%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udev/moc_udevdevice.cpp.o
[ 29%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udev/moc_udevdeviceinterface.cpp.o
[ 29%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udev/moc_udevgenericinterface.cpp.o
[ 29%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udev/moc_udevmanager.cpp.o
[ 29%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udev/moc_udevportablemediaplayer.cpp.o
[ 30%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/shared/moc_udevqt.cpp.o
[ 30%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/moc_halblock.cpp.o
[ 30%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/moc_halcdrom.cpp.o
[ 30%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/moc_haldevice.cpp.o
[ 30%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/moc_haldeviceinterface.cpp.o
[ 30%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/moc_halgenericinterface.cpp.o
[ 30%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/moc_halmanager.cpp.o
[ 30%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/moc_halopticaldisc.cpp.o
[ 31%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/moc_halportablemediaplayer.cpp.o
[ 31%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/moc_halstorageaccess.cpp.o
[ 31%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/moc_halstorage.cpp.o
[ 31%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/moc_udisksblock.cpp.o
[ 31%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/hal/moc_halvolume.cpp.o
[ 31%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/moc_udisksdevicebackend.cpp.o
[ 31%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/moc_udisksdevice.cpp.o
[ 31%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/moc_udisksdeviceinterface.cpp.o
[ 32%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/moc_udisksgenericinterface.cpp.o
[ 32%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/moc_udisksmanager.cpp.o
[ 32%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/moc_udisksopticaldisc.cpp.o
[ 32%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/moc_udisksopticaldrive.cpp.o
[ 32%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/moc_udisksstorageaccess.cpp.o
[ 32%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/moc_udisksstoragedrive.cpp.o
[ 32%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/moc_udisksstoragevolume.cpp.o
[ 33%] Building CXX object 3rdparty/solid-lite/CMakeFiles/solidlite.dir/backends/udisks2/dbus/moc_manager.cpp.o
[ 33%] Linking CXX static library libsolidlite.a
[ 33%] Built target solidlite
Makefile:149: recipe for target 'all' failed
make: *** [all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...
```

**Comment by CDrummond** [comment]
Your system is missing: libavutil/audioconvert.h

**Comment by ms6676749** [comment]
Which is missing from Arch's latest package, hmm: https://www.archlinux.org/packages/extra/x86_64/ffmpeg/files/

It appears that audioconvert.h header has been deprecated: http://permalink.gmane.org/gmane.comp.video.libav.scm/17242

**Comment by goetzc** [comment]
Deprecated in 2012 and no longer available since ffmpeg version 1.3

**Comment by CDrummond** [comment]
Oops :-( I'll update the code to no longer use this header!

**Comment by CDrummond** [comment]
Fixed in master and 2.0 branch. Will release 2.0.1 soon.
